src/components/Chart/model.ts

```diff
@@ -68,11 +68,11 @@ export interface PieChartDataPoint {
 
 interface RangeAreaMask<XDomain> {
   show?: boolean
-  showDots?: boolean
-  showDotsOffset?: XDomain
+  hideDots?: boolean
+  hideDotsStart?: XDomain
```

Comment:
```suggestion
  hideDots?: boolean | { from?: XDomain, to?: XDomain }
```
I think it's clearer this way

Reply:
OK... So i think it should be an prop of this new object too, right? Something like `overlay: boolean`